src/index.html

```diff
@@ -11,12 +11,53 @@
       content="ie=edge"
     />
     <title>Moyo header</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;700&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a class="img-wraper">
+        <img
+          src="./images/logo.png"
+          alt="Logotype"
+        />
```

Comment:
The `alt` attribute should describe the image content. It improves your page SEO and helps screen readers. `Logotype` does not provide enough information about the content of the image. It should be more descriptive.